boards: arm: Add PAN1783 evaluation board

pideu-sj opened the pull request:
The PAN1783 evaluation board is a development tool for the nRF5340 from Nordic Semiconductor.

pideu-sj committed:
boards: arm: Add PAN1783 evaluation board

The PAN1783 evaluation board is a development tool for the nRF5340
from Nordic Semiconductor.

Signed-off-by: Steffen Jahnke <steffen.jahnke@eu.panasonic.com>

gmarull commented on boards/arm/pan1783_evb/pan1783_evb_cpuapp_defconfig:
```diff
@@ -0,0 +1,30 @@
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC=y
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL=n
+
+CONFIG_PINCTRL=y
```
no longer needed, it's already selected by drivers

pideu-sj replied:
Okay, could you maybe describe that? All of them can be removed? Or the whole file? 

I've tried to remove some of the configurations and tried to build samples/hello_world, but get build errors now or sample is not working (no uart output, clock is not working,...)? 

Sorry I'm still not that familiar with these topics. Thanks.

pideu-sj commented:
Hello,
has maybe someone an idea why the none secure twister runs are failing because of "**Unsupported TFM_PLATFORM**" error? What does that mean? Can't find the reason for that right now.

The same for the compliance check. When I'm looking into the signed off: "**Signed-off-by: Steffen Jahnke <steffen.jahnke@eu.panasonic.com>"** is looking good to me?

pideu-sj commented:
Hi @gmarull 

the compliance check says:
```
author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) needs to match one of the signed-off-by entries.
author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) does not follow the syntax: First Last <email>.
```
so on first time I accidentally used another pattern on my sign off. But then I did a rebase and updated it. See the commit message:
```
boards: arm: Add PAN1783 evaluation board

The PAN1783 evaluation board is a development tool for the nRF5340
from Nordic Semiconductor.

Signed-off-by: Steffen Jahnke <steffen.jahnke@eu.panasonic.com>
```

Why does the compliance check still mark that as an issue? Is there a bug?

gmarull commented:
> Hi @gmarull
> 
> the compliance check says:
> 
> ```
> author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) needs to match one of the signed-off-by entries.
> author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) does not follow the syntax: First Last <email>.
> ```
> 
> so on first time I accidentally used another pattern on my sign off. But then I did a rebase and updated it. See the commit message:
> 
> ```
> boards: arm: Add PAN1783 evaluation board
> 
> The PAN1783 evaluation board is a development tool for the nRF5340
> from Nordic Semiconductor.
> 
> Signed-off-by: Steffen Jahnke <steffen.jahnke@eu.panasonic.com>
> ```
> 
> Why does the compliance check still mark that as an issue? Is there a bug?

`Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>` != `Steffen Jahnke <steffen.jahnke@eu.panasonic.com>`. Make sure you sign using `git commit -s`

pideu-sj commented:
> > Hi @gmarull
> > the compliance check says:
> > ```
> > author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) needs to match one of the signed-off-by entries.
> > author email (Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>) does not follow the syntax: First Last <email>.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > so on first time I accidentally used another pattern on my sign off. But then I did a rebase and updated it. See the commit message:
> > ```
> > boards: arm: Add PAN1783 evaluation board
> > 
> > The PAN1783 evaluation board is a development tool for the nRF5340
> > from Nordic Semiconductor.
> > 
> > Signed-off-by: Steffen Jahnke <steffen.jahnke@eu.panasonic.com>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Why does the compliance check still mark that as an issue? Is there a bug?
> 
> `Jahnke, Steffen <Steffen.Jahnke@eu.panasonic.com>` != `Steffen Jahnke <steffen.jahnke@eu.panasonic.com>`. Make sure you sign using `git commit -s`

Yes, I know. That was the reason why I did a rebase and updated it to match the guidelines. When you look on my current commit message, the sign-off looks like it should. Or am I somehow wrong here? It looks the same as on other commit messages from others.
